player/command.c

```diff
@@ -7127,6 +7127,11 @@ static void cmd_file_dialog(void *p)
     struct mp_cmd_ctx *cmd = p;
     struct MPContext *mpctx = cmd->mpctx;
 
+    if (!mp_input_is_interactive_cmd(cmd->cmd)) {
+        cmd->success = false;
+        return;
+    }
```

Comment:
Since this is naturally already defined to be always interactive, isn't this just redundant?

Reply:
Yes, fixed.